umap/static/umap/js/umap.layer.js

```diff
@@ -1013,6 +1013,12 @@ L.U.DataLayer = L.Evented.extend({
         this.map.post(this.getSaveUrl(), {
             data: formData,
             callback: function (data, response) {
+                // Response contains geojson only if save has conflicted and conflicts have
+                // been resolved. So we need to reload to get extra data (saved from someone else)
+                if (data.geojson) {
+                    this.clear();
+                    this.fromGeoJSON(data.geojson);
```

Comment:
We may also reload the whole map in this case, to lower the possible conflicts in other layers. But that would be a bit more aggressive for the user to have whole page reloaded.

Reply:
Other option instead of returning the geojson: make a 302 to the "normal" GET